### ISSUE TITLE
Error 10426 when applying 99% coupon on a $89 item

### ISSUE THREAD

**Issue by spraveenitpro** [woocommerce/woocommerce-gateway-paypal-express-checkout#557]
**Description:**

We are seeing PayPal Error 10426 when applying 99% coupon on a $89 item. 

![](https://cld.wthms.co/9ORpVS+)
[Direct Link](https://cld.wthms.co/9ORpVS+)

**Steps to recreate the issue**

1. Create a discount coupon for 99%
2. Create a product for $89 [no tax and free shipping]
3. Checkout and attempt to pay with PayPal checkout

**Demo**:

![](https://cld.wthms.co/kcRCKN+)
[Direct Link](https://cld.wthms.co/kcRCKN+)

**Logs**:

```
WC_Gateway_PPEC_Client::_process_response: acknowleged response body: Array ( [TIMESTAMP] => 2019-04-15T22:30:45Z [CORRELATIONID] => 33d5a5ce8dda0 [ACK] => Failure [VERSION] => 120.0 [BUILD] => 52490036 [L_ERRORCODE0] => 10426 [L_SHORTMESSAGE0] => Transaction refused because of an invalid argument. See additional error messages for details. [L_LONGMESSAGE0] => Item total is invalid. [L_SEVERITYCODE0] => Error )
```
@budzanowski

**Comment by menakas** [comment]
@spraveenitpro 

I tried with the exact same setttings: 
Simple Product with regular price $89.99(as shown in your image).
Applied a Percentage discount coupon of 99%.
Cart total is $0.90

The payment was succesful and the order was created without any issues.
https://d.pr/i/vQqnjx
I am closing this now. If there are any specifics that I've missed, please feel free to reopen the issue.

**Comment by menakas** [comment]
Duplicate of #633 fixed with #634
